src/dash-playlist-loader.js

```diff
@@ -901,4 +906,31 @@ export default class DashPlaylistLoader extends EventTarget {
 
     this.trigger('loadedplaylist');
   }
+
+  /**
+   * Takes eventstream data from a parsed DASH manifest and adds it to the metadata text track.
+   *
+   * @param {manifest} newMain the newly parsed manifest
+   */
+  addEventStreamToMetadataTrack_(newMain) {
+    // Only add new event stream metadata if we have a new manifest.
+    if (newMain && this.mainPlaylistLoader_.main.eventStream) {
+      createMetadataTrackIfNotExists(this.inbandTextTracks_, 'EventStream', this.tech_);
+
+      // convert EventStream to ID3-like data.
+      this.mainPlaylistLoader_.main.eventStream.forEach((eventStreamNode) => {
+        const metadataArray = [{
+          cueTime: eventStreamNode.start,
+          frames: [{ data: eventStreamNode.messageData || eventStreamNode.value }]
```

Comment:
I am not super familiar with how `messageData` and `value` differ in the actual manifest file, but could we include an entry in the manifest so we ensure we test both of these?

Reply:
I actually just re-read the spec and have a little refactoring to do here regarding the expected node return values. Will update the PR shortly.

Reply:
Per my previous comment, I'm removing `eventStream.value` from the frames object. This will actually be addressed downstream in the `mpd-parser` see: https://github.com/videojs/mpd-parser/pull/171